src/query/expression/src/values.rs

```diff
@@ -1597,6 +1655,14 @@ impl Column {
                     offsets: offsets.into(),
                 }))
             }
+            DataType::Bitmap => BitmapType::from_data((0..len).map(|_| {
+                let rng = SmallRng::from_entropy();
```

Comment:
Need to generate a few random numbers and insert them into a `RoaringBitmap` object and then converted it to binary format using the function `serialize_into`. The random values generated directly may not be a valid `RoaringBitmap`.